src/app/[bookId]/page.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { useState, useEffect } from 'react'
+
+import BooksAPI from '@/api/books-api'
+import BookInfo from '@/app/[bookId]/components/book-info'
+import BookLikeButton from '@/app/[bookId]/components/book-like-button'
+import BookOpinionTabs from '@/app/[bookId]/components/book-opinion-tabs'
+import type { Book } from '@/types/book'
+
+interface BookProps {
+  params: { bookId: string }
+}
+
+const Book = ({ params }: BookProps) => {
+  const [book, setBook] = useState<Book | null>(null)
+
+  const fetchBook = async () => {
+    const fetchedBook = await BooksAPI.getBookById(params.bookId)
+    setBook(fetchedBook)
```

Comment:
```suggestion
    try {
      const fetchedBook = await BooksAPI.getBookById(params.bookId)
      setBook(fetchedBook)
    } catch {
      setBook(null)
    }
```

you need catch errors

Reply:
Fixed